src/_spec/get.js

```diff
@@ -50,6 +50,20 @@ describe('get', () => {
     assert.equal(date, 2)
   })
 
+  it('should return the week for a monday', () => {
+    const input = new Date('2015-03-04 09:08:05.023')
```

Comment:
Could we use a Monday and Sunday that are next to each other or in the same week so that it is clear when reading the tests how this works?